src/hotspot/cpu/riscv/globals_riscv.hpp

```diff
@@ -115,6 +115,7 @@ define_pd_global(intx, InlineSmallCode,          1000);
   product(bool, UseZtso, false, EXPERIMENTAL, "Assume Ztso memory model")        \
   product(bool, UseZihintpause, false, EXPERIMENTAL,                             \
           "Use Zihintpause instructions")                                        \
+  product(bool, UseZvbb, false, "Use Zvbb instructions")                         \
```

Comment:
That'll conflict with https://github.com/openjdk/jdk/pull/19065, but same, we'd want to have `EXPERIMENTAL`

Reply:
Yes, this will be fixed after https://github.com/openjdk/jdk/pull/19065 by merging from master.

Reply:
Now, merged/fixed.